impeller/renderer/render_target.cc

```diff
@@ -195,6 +214,17 @@ RenderTarget& RenderTarget::SetStencilAttachment(
   return *this;
 }
 
+ColorAttachment RenderTarget::GetColor0() const {
+  if (color0_.has_value()) {
+    return color0_.value();
+  }
+  return ColorAttachment{};
+}
+
+bool RenderTarget::HasColor0() const {
```

Comment:
Same thing here.  Let's just make `HasColorAttachment(0)` look into this.

Reply:
Done